backend/src/test/resources/testResultUpload/test-results-upload-valid-as-fhir.ndjson

```diff
@@ -0,0 +1,6 @@
http://terminology.hl7.org/CodeSystem/v2-0119","code":"RE"}]}},{"url":"https://reportstream.cdc.gov/fhir/StructureDefinition/order-effective-date","valueDateTime":"2021-12-20T14:00:00-06:00"}],"status":"completed","intent":"order","code":{"coding":[{"system":"http://loinc.org","code":"95422-2"}]},"subject":{"reference":"Patient/4567"},"requester":{"reference":"PractitionerRole/5db534ea-5e97-4861-ba18-d74acc46db15"},"performer":[{"reference":"Organization/12345000-0000-0000-0000-000000000000"}],"supportingInfo":[{"reference":"Observation/a65bbde7-854d-3151-b46b-731badba996b"}]}},{"fullUrl":"Device/5db534ea-5e97-4861-ba18-d74acc46db15","resource":{"resourceType":"Device","id":"5db534ea-5e97-4861-ba18-d74acc46db15","manufacturer":"Acme","deviceName":[{"name":"ID NOW","type":"model-name"}]}},{"fullUrl":"PractitionerRole/5db534ea-5e97-4861-ba18-d74acc46db15","resource":{"resourceType":"PractitionerRole","id":"5db534ea-5e97-4861-ba18-d74acc46db15","practitioner":{"reference":"Practitioner/1013012657"},"organization":{"reference":"Organization/5db534ea-5e97-4861-ba18-d74acc46db15"}}},{"fullUrl":"Organization/07640c5d-87cd-488b-9343-a226c5166539","resource":{"resourceType":"Organization","id":"07640c5d-87cd-488b-9343-a226c5166539","name":"SimpleReport"}},{"fullUrl":"Observation/5db534ea-5e97-4861-ba18-d74acc46db15","resource":{"resourceType":"Observation","id":"5db534ea-5e97-4861-ba18-d74acc46db15","status":"final","code":{"coding":[{"system":"http://loinc.org","code":"94534-5"}],"text":"COVID-19"},"subject":{"reference":"Patient/4567"},"issued":"2021-12-23T20:00:00.000Z","performer":[{"reference":"Organization/12345000-0000-0000-0000-000000000000"}],"valueCodeableConcept":{"coding":[{"system":"http://snomed.info/sct","code":"260373001","display":"Detected"}]},"interpretation":[{"coding":[{"system":"http://terminology.hl7.org/CodeSystem/v2-0078","code":"A","display":"Abnormal"}]}],"method":{"extension":[{"url":"https://reportstream.cdc.gov/fhir/StructureDefinition/testkit-name-id","valueCoding":{"code":"BOOMX2"}},{"url":"https://reportstream.cdc.gov/fhir/StructureDefinition/equipment-uid","valueCoding":{"code":"543212134"}}],"coding":[{"display":"ID NOW"}]},"specimen":{"reference":"Specimen/5db534ea-5e97-4861-ba18-d74acc46db15"},"device":{"reference":"Device/5db534ea-5e97-4861-ba18-d74acc46db15"}}},{"fullUrl":"Observation/a65bbde7-854d-3151-b46b-731badba996b","resource":{"resourceType":"Observation","id":"a65bbde7-854d-3151-b46b-731badba996b","identifier":[{"use":"official","type":{"coding":[{"system":"http://loinc.org","code":"81959-9","display":"Public health laboratory ask at order entry panel"}]}}],"status":"final","code":{"coding":[{"system":"http://loinc.org","code":"95419-8","display":"Has symptoms related to condition of interest"}],"text":"Has symptoms related to condition of interest"},"subject":{"reference":"Patient/4567"},"valueCodeableConcept":{"coding":[{"system":"http://terminology.hl7.org/ValueSet/v2-0136","code":"N","display":"No"}]}}}]}
```

Comment:
This file contains all the accepted values for test results in ndjson FHIR format

Positive
Negative
Detected
Not Detected
Invalid Result
snomed value (260373001) has an added `detected` as display value in the FHIR